lrlex/src/lib/ctbuilder.rs

```diff
@@ -78,11 +78,47 @@ pub enum RustEdition {
     Rust2021,
 }
 
+use proc_macro2::TokenStream;
+use quote::{ToTokens, TokenStreamExt};
```

Comment:
These should probably go with the other `use`s at the top of the file.

Reply:
Should be fixed in [f6d33f4](https://github.com/softdevteam/grmtools/pull/494/commits/f6d33f49b5d685aa9952122f6640f80433926fdc)